pennylane_ls/FermionDevice.py

```diff
@@ -154,13 +154,14 @@ def check_job_status(self, job_id):
 
     def wait_till_done(self, job_id):
         while True:
-            time.sleep(2)
+            time.sleep(0.1)
             job_status = self.check_job_status(job_id)
             if job_status == "DONE":
                 break
+            elif job_status == "INITIALIZING":
+                continue
             else:
-                pass
-                # print(job_status)
+                raise DeviceError(job_status)
```

Comment:
I would raise error such that error detail is presented to the user. I already did it for all 3 devices in #48 .

Reply:
Yes, what you did in #48 is much better than this.